submariner-operator/issue-359-subctl-info-status.md

```diff
@@ -0,0 +1,116 @@
+#Issue 359: Add more information to subctl info cmd
+
+## Summary
+Current cmd:  
+* subctl info  
+    Discovered network details:  
+         Network plugin:  weave-net  
+         Service CIDRs: [100.92.0.0/16]  
+         Cluster CIDRs: [10.242.0.0/16]
+
+
+## Motivation
+To supply more information on current deployment status
+
+## Proposal
+* _subctl status network_  
+    Discovered network details:  
+         Network plugin:  weave-net  
+         Service CIDRs: [100.92.0.0/16]  
+         Cluster CIDRs: [10.242.0.0/16]  
+
+
+* _subctl status gateways_
+    * endpoint:  
```

Comment:
What about something like:

```
$ subctl status gateways
NODE       HA-STATUS       CONNECTIONS    SUMMARY
worker1    active          3/3            All connections OK
worker2    passive         0/0                                

^  this would return exit code != 0 if any connection is in error or connecting

$ subctl status connections
GATEWAY  CLUSTER    REMOTE-IP    DRIVER     SUBNETS                         STATUS
worker1  cluster2   172.17.0.10  libreswan  100.93.0.0/16,  10.243.0.0/16   connected
worker1  cluster3   172.17.0.11  libreswan  100.94.0.0/16,  10.244.0.0/16   connected

^ this would return exit code !=0 if any connection is in error or connecting

 ```

We can probably ignore the local endpoint part and expose that somewhere else.